services/portal/studyViewer/studyViewerTasks.js

```diff
@@ -5,7 +5,7 @@ const I = actor();
 module.exports = {
 
   goToStudyViewerPage() {
-    I.amOnPage(studyViewerProps.path); /// study-viewer/clinical_trials
+    I.amOnPage(studyViewerProps.path); // study-viewer/clinical_trials
     I.waitForVisible(studyViewerProps.studyViewerDivClass, 5); // .study-viewer
     I.saveScreenshot('study_viewer_page.png');
```

Comment:
i see 6 "saveScreenshot"s, do you usually keep them in the code or is it just for debugging while writing the tests? if it's the former, some of them could use more explicit filenames and  if it's the latter, could you remove them

Reply:
"saveScreenshot" code will help us debug the test, if it fails in the pipeline. We might have to do it now or later for better debugging the failing test. So I put in there where there are chances of tests failing. So it will help us and we could debug the failing test.